packages/extension-ui/src/components/MenuItem.tsx

```diff
@@ -17,23 +17,27 @@ function MenuItem ({ children, className = '', title }: Props): React.ReactEleme
   return (
     <div className={className}>
       {title && (
-        <Title className='itemTitle'>{title}</Title>
+        <div className='itemTitle'>{title}</div>
       )}
       {children}
     </div>
   );
 }
 
-export default styled(MenuItem)`
+export default styled(MenuItem)(({ theme }: ThemeProps) => `
   min-width: 13rem;
   padding: 0 16px;
   max-width: 100%;
 
   > .itemTitle {
-    margin: 0;
-  }
-
-  &+& {
```

Comment:
same here, I removed it entirely, made no difference I could see. This should apply when we have a `menuItem` directly nested into a `menuIdem`, but this is never the case (any more?) in practice.